include/z64save.h

```diff
@@ -240,16 +240,16 @@ typedef struct SaveContext {
     /* 0x3F3F */ u8 unk_3F3F;                           // "flash_flag"
     /* 0x3F40 */ SaveOptions options;
     /* 0x3F46 */ u16 unk_3F46;                          // "NottoriBgm"
-    /* 0x3F48 */ u8 unk_3F48;                           // "fade_go"
+    /* 0x3F48 */ u8 cutsceneTransitionControl;          // "fade_go"
```

Comment:
(Wonder if this is boolean)

Reply:
`u8 cutsceneTransitionControl; // context dependent usage: can either trigger a delayed fade or control fill alpha`
but MM's z_demo no longer has:
`gSaveContext.cutsceneTransitionControl = 255.0f - (155.0f * temp);`
So maybe? Guess we'll have to wait for `z_play` to find out

Reply:
the original name of fade_go noted there lines up more with the delayed fade case:
```
case TRANS_MODE_INSTANCE_WAIT:
    if (gSaveContext.cutsceneTransitionControl != 0) {
        this->transitionMode = TRANS_MODE_INSTANCE_RUNNING;
    }
    break;
    ```
oot uses the same variable to control the fade in in links house during the intro, so it has to have the dumb generic name.
should of course wait for the related z_play code to confirm, but it seems likely that it can be treated as the trigger for a delayed fade from a cutscene